assets/src/dashboard/app/api/useStoryApi.js

```diff
@@ -76,6 +76,7 @@ const useStoryApi = (dataAdapter, { editStoryURL, storyApi }) => {
       }
 
       const query = {
+        _embed: 'author',
```

Comment:
```suggestion
        _embed: 1
```
Embed doesn't support just embedding one field. It is all or nothing.

Reply:
No, it **does** support that. See https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/#_embed which is linked to from the PR description

Reply:
I am wrong. Thanks for the feedback.